lld/COFF/Driver.cpp

```diff
@@ -365,6 +380,24 @@ bool LinkerDriver::isDecorated(StringRef sym) {
          (!ctx.config.mingw && sym.contains('@'));
 }
 
+static LLVM_THREAD_LOCAL bool executingFirstQueue;
```

Comment:
Rather than TLS for every thread, can this be a LinkerDriver field and these helpers private methods?